cinema/views.py

```diff
@@ -0,0 +1,48 @@
+from django.http import HttpResponse
+from django.views.decorators.csrf import csrf_exempt
+from rest_framework.decorators import api_view
+from rest_framework.parsers import JSONParser
+from rest_framework.response import Response
+from rest_framework import status
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    if request.method == "POST":
+        serializer = MovieSerializer(data=request.data)
+
+        if serializer.is_valid():
+            serializer.save()
+            return Response(serializer.data, status=status.HTTP_201_CREATED)
+
+
+@csrf_exempt
```

Comment:
I don't know if you need to use the `@csrf_exempt` decorator here, but I think if you implement it here, you should also apply it in the view function above.
By the way, if we use the `@api_view` decorator, then there is no need to protect against `csrf-attack` because our decorator wraps `APIView` and implements it in the `as_view` method.
But maybe I'm wrong. I don't think you need to change it now, I think the mentor will give you better explanations.

Reply:
![image](https://user-images.githubusercontent.com/110720407/202170669-5473e072-27af-4523-883c-200b37d229df.png)

Reply:
thanks! I decided to just use api_view here:)